File: Beecrowd/1 - Beginner/1435 - 1985/1541 - Building Houses/1541_BuildingHouses.py
import math

while True:
    n = input().split()
    if n == ['0']:
        break
    
    size1, size2, maxAllowed = n
    size1, size2, maxAllowed = int(size1), int(size2), int(maxAllowed)
    
    area = size1 * size2
    size3 = math.sqrt((area * 100) / maxAllowed)
    
    print(int(size3))

File: Beecrowd/1 - Beginner/1435 - 1985/1541 - Building Houses/1541_BuildingHouses.c
#include <stdio.h>
#include <math.h>

int main() {
    int size1, size2, maxAllowed;

    while (scanf("%d", &size1) == 1 && size1 != 0) {
        scanf("%d %d", &size2, &maxAllowed);

        int area = size1 * size2;
        int side = sqrt((area * 100) / maxAllowed);

        printf("%d\n", side);
    }

    return 0;
}
